src/cryptoadvance/specter/managers/node_manager.py

```diff
@@ -29,7 +29,9 @@ def __init__(
         internal_bitcoind_version="",
         data_folder="",
     ):
-        self.nodes = {}
+        self.nodes = (
+            {}
+        )  # Sth. like: {'nigiri_regtest': <Node name=Nigiri regtest fullpath=...>, 'default': <Node name=Bitcoin Core fullpath=...>}
```

Comment:
Why the single brackets? I don't understand that. The example does not help with this?!

Reply:
They are from black because of the comment added.

Reply:
Resolved in https://github.com/cryptoadvance/specter-desktop/pull/1974/commits/b19cb6d5bcc5dd37bf921d6b92a586135683f83b